### ISSUE TITLE
Intermittently getting the error 'cannot forward the request No System TLS'

### ISSUE THREAD

**Issue by gsaimadhu** [zalando/zalenium#568]
**Please make sure that you provide enough information for us to help you with this issue. Thank you!**
We have around 500 tests to run, starting 11 threads in parallel.

# Zalenium Image Version(s):  
#docker images | grep zalenium
dosel/zalenium                                                     latest              6a8d6e5f89b4        3 days ago          792MB
#docker images | grep selenium
elgalu/selenium                                                    latest              be74142e235f        21 hours ago        1.37GB
# Docker Version:  

#docker version
Client:
 Version:      17.05.0-ce
 API version:  1.29
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:06:25 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.05.0-ce
 API version:  1.29 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:06:25 2017
 OS/Arch:      linux/amd64
 Experimental: false

# OS: 
centos7
uname -a
Linux apim-centos7-agent28 3.10.0-693.21.1.el7.x86_64 #1 SMP Wed Mar 7 19:03:37 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

# Docker Command to start Zalenium:  

docker run --rm -t --name zalenium -p 4444:4444      -v /var/run/docker.sock:/var/run/docker.sock       -v /tmp/videos:/home/seluser/videos   --add-host "t1.l7tech.com:$ipa" --add-host "tenant2.l7tech.com:$ipa" --add-host "tenant3.l7tech.com:$ipa"  --add-host "t4.l7tech.com:$ipa"  --add-host "t12345.l7tech.com:$ipa"   --add-host "t6.l7tech.com:$ipa"   --add-host "t7.l7tech.com:$ipa"   --add-host "t8.l7tech.com:$ipa"   --add-host "tnt9.l7tech.com:$ipa"   --add-host "t10.l7tech.com:$ipa"   --add-host "televen.l7tech.com:$ipa"  --privileged dosel/zalenium start  --desiredContainers 12 --maxDockerSeleniumContainers 30 --maxTestSessions 100  > ZaleniumStartLogs.log &

## Expected Behavior -
No error from browser
## Actual Behavior -
Intermittently getting No System TLS.
[ZaleniumStartLogs.log](https://github.com/zalando/zalenium/files/1994312/ZaleniumStartLogs.log)

**Comment by amalsgit** [comment]
Looks like this is the issue which I am also facing.

Zalenium Image Version(s):
Always the latest. I pull the latest before container creation as part of Jenkins script

OS:
Centos7

Docker Command to start Zalenium:

`sudo docker run -d --name zalenium -p 4444:4444 \\
        -v /var/run/docker.sock:/var/run/docker.sock \\
        -v /tmp/videos:/home/seluser/videos \\
        -v /home/centos/workspace/${JOB_NAME}/robotTest/robot-web-tests/results/robotDownloads:/tmp/node/home/seluser/robotDownloads \\
        --privileged dosel/zalenium start --desiredContainers 4 --maxDockerSeleniumContainers 10 --maxTestSessions 50 --videoRecordingEnabled false`

Docker logs: 
[docker_log.txt](https://github.com/zalando/zalenium/files/1994606/docker_log.txt)

**Comment by pearj** [comment]
Duplicate of #491 closing
